inc/Engine/Media/AboveTheFold/Activation/Activation.php

```diff
@@ -0,0 +1,55 @@
+<?php
+declare(strict_types=1);
+
+namespace WP_Rocket\Engine\Media\AboveTheFold\Activation;
+
+use WP_Rocket\Engine\Activation\ActivationInterface;
+use WP_Rocket\Engine\Media\AboveTheFold\WarmUp\Controller;
+use WP_Rocket\Engine\Common\JobManager\Managers\ManagerInterface;
+
+class Activation implements ActivationInterface {
+	/**
+	 * WarmUp controller
+	 *
+	 * @var Controller
+	 */
+	private $controller;
+
+	/**
+	 * Above the fold Job Manager.
+	 *
+	 * @var ManagerInterface
+	 */
+	private $manager;
+
+	/**
+	 * Instantiate class.
+	 *
+	 * @param Controller       $controller Controller instance.
+	 * @param ManagerInterface $manager Above the fold Job Manager.
+	 */
+	public function __construct( Controller $controller, ManagerInterface $manager ) {
```

Comment:
Above the fold manager doesn't exist anymore as part of https://github.com/wp-media/wp-rocket/issues/6461, does it impact here ?